src/lib/poracleMessage/commands/egg.js

```diff
@@ -108,7 +107,6 @@ exports.run = async (client, msg, args, options) => {
 									...toInsert,
 									uid: existing.uid,
 								})
-								insert.splice(i, 1)
```

Comment:
I see the change from updates to deletes & inserts, which works on the database side but the user feedback will be duplicated (ie have an entry as 'New' and another as 'updated')

Reply:
worked around it in abca4fe. 
I still believe that a delete + insert is better than an update call for each mon